install/cupy_builder/_features.py

```diff
@@ -422,7 +422,7 @@ def get_features(ctx: Context) -> Dict[str, Feature]:
         features = [
             CUDA_cuda(ctx),
             _from_dict(CUDA_cusolver, ctx),
-            _from_dict(CUDA_cudnn, ctx),
+            # _from_dict(CUDA_cudnn, ctx),
```

Comment:
Note this is needed for me locally (my system has an incompatible cudnn version), should revert the corresponding commit before merging if this goes in.